chia/wallet/trading/offer.py

```diff
@@ -683,12 +684,12 @@ def from_bech32(cls, offer_bech32: str) -> Offer:
     # We basically hijack the SpendBundle versions for most of it
     @classmethod
     def parse(cls, f: BinaryIO) -> Offer:
-        parsed_bundle = SpendBundle.parse(f)
+        parsed_bundle = parse_rust(f, SpendBundle)
         return cls.from_bytes(bytes(parsed_bundle))
 
     def stream(self, f: BinaryIO) -> None:
-        as_spend_bundle = SpendBundle.from_bytes(bytes(self))
-        as_spend_bundle.stream(f)
+        spend_bundle_bytes = self.to_spend_bundle().to_bytes()
```

Comment:
Is there a reason to not use `bytes(self)` here?

Reply:
not really. do you think that would be preferable?